src/nunjucks/moduk/components/checkboxes/__examples__/with-conditional-reveal.njk

```diff
@@ -0,0 +1,82 @@
+{% from "moduk/components/checkboxes/macro.njk" import modukCheckboxes %}
+
+{# TODO: Switch to modukInput when it's been added #}
+{% from "govuk/components/input/macro.njk" import govukInput %}
```

Comment:
Had to use `govukInput` here as we don't have `modukInput` yet – we'll need to switch over once we've added `modukInput`.

Reply:
I'll have to do the same as radios is in dev now